modules/polling/helpers/hasMkrRequiredForGaslessVotingCheck.ts

```diff
@@ -0,0 +1,15 @@
+import { SupportedNetworks } from 'modules/web3/constants/networks';
+import { getMKRVotingWeight, MKRVotingWeightResponse } from 'modules/mkr/helpers/getMKRVotingWeight';
+import { MIN_MKR_REQUIRED_FOR_GASLESS_VOTING } from 'modules/polling/polling.constants';
+
+export async function hasMkrRequiredForGaslessVotingCheck(
+  voter: string,
```

Comment:
we can refactor this function to "hasMKRRequiredWeight" and pass the amount as a parameter, this way we can reuse to normal users also, passing "0.0" to check if they have more than 0 mkr